package.json

```diff
@@ -44,7 +44,7 @@
   "homepage": "https://github.com/morpho-labs/ethers-multicall#readme",
   "dependencies": {
     "dataloader": "^2.1.0",
-    "ethers": "^5.7.0",
+    "ethers": "^5.0.0",
```

Comment:
why downgrading?

Reply:
Because the required version is only v5, we don't have requirement on minor versions or patches